src/routes/_public/guitar-tab/$slug.tsx

```
@@ -0,0 +1,282 @@
+
+  if (!data) return null;
+
+  const { tab, relatedTabs } = data;
+  const displayTitle = tab.title || tab.fileName;
+  const coverUrl = tab.coverKey
+    ? `/images/${tab.coverKey}?w=300&h=300&fit=cover`
+    : null;
+  const fileUrl = `/images/${tab.key}?original=true`;
+
+  // 查看器状态
+  const [viewerOpen, setViewerOpen] = useState(false);
```

Biome (2.4.4)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)